src/core/census/grpc_filter.c

```diff
@@ -115,8 +115,8 @@ static void server_mutate_op(grpc_call_element *elem,
 static void server_start_transport_op(grpc_exec_ctx *exec_ctx,
                                       grpc_call_element *elem,
                                       grpc_transport_stream_op *op) {
-  call_data *calld = elem->call_data;
-  GPR_ASSERT((calld->op_id.upper != 0) || (calld->op_id.lower != 0));
+  /* call_data *calld = elem->call_data;
```

Comment:
either remove completely or add a TODO explaining why the block has been (temporarily) commented out.